src/JWT/Jwk/JwtWebKeysCollection.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+using System.Linq;
+using JWT.Serializers;
+
+namespace JWT.Jwk
+{
+    public class JwtWebKeysCollection : IJwtWebKeysCollection
```

Comment:
Please `seal` all classes not intended to be overridden

Reply:
Fixed. This class was the only one not sealed.